src/stacked_borrows.rs

```diff
@@ -607,29 +613,30 @@ pub trait EvalContextExt<'a, 'mir, 'tcx: 'a+'mir>: crate::MiriEvalContextExt<'a,
             MemoryKind::Stack => {
                 // New unique borrow. This `Uniq` is not accessible by the program,
                 // so it will only ever be used when using the local directly (i.e.,
-                // not through a pointer).  IOW, whenever we directly use a local this will pop
-                // everything else off the stack, invalidating all previous pointers
-                // and, in particular, *all* raw pointers.  This subsumes the explicit
+                // not through a pointer). That is, whenever we directly use a local, this will pop
+                // everything else off the stack, invalidating all previous pointers,
+                // and in particular, *all* raw pointers. This subsumes the explicit
                 // `reset` which the blog post [1] says to perform when accessing a local.
                 //
-                // [1] https://www.ralfj.de/blog/2018/08/07/stacked-borrows.html
+                // [1]: <https://www.ralfj.de/blog/2018/08/07/stacked-borrows.html>
                 this.machine.stacked_borrows.increment_clock()
             }
             _ => {
-                // Nothing to do for everything else
+                // Nothing to do for everything else.
                 return Borrow::default()
             }
         };
-        // Make this the active borrow for this allocation
-        let alloc = this.memory_mut().get_mut(id).expect("This is a new allocation, it must still exist");
+        // Make this the active borrow for this allocation.
+        let alloc = this.memory_mut().get_mut(id)
+                        .expect("this is a new allocation; it must still exist");
```

Comment:
keep the old code

Reply:
Isn't it good to line-wrap?

Reply:
I like 120 chars :laughing: 

But line wrapping is often not a problem (e.g. here), but if done, it should be using block indent, not visual indent.

Reply:
I believe the fmt-rfcs decision was either fit all in one line or block indent with one method call per line. (Never use alignment.)

Reply:
@oli-obk So do I hah. What should we say here though. Always wrap at 120? Always at 100? Always at one of these except for debug comments?

Reply:
I don't care too much when one decides to wrap. But if the code is wrapped, it should be block indented. Also I prefer to only change code when actually modifying something in the same lines.

Reply:
This was not addressed.